### PR TITLE
Update readme with build instructions for the rev3

### PR DESCRIPTION
**Pull request by jeremydaily**
Also added the gerber files for Rev 4 with a bill of materials.

### CODE REVIEW

**Comment by jeremydaily** [comment]
@BenGardiner I'm thinking of having these ready to go for the CyberTruck Challenge.

**Review by BenGardiner — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -4,6 +4,64 @@ or better yet, "can-in-the-middle" is system using the TU Truck Cape, a BeagleBo
...
+#### Using Python
+In the `os.system` command enables execution of these commands.
+
+```
+debian@beaglebone:~$ python3
+Python 3.5.3 (default, Jan 19 2017, 14:11:04)
+[GCC 6.3.0 20170118] on linux
+Type "help", "copyright", "credits" or "license" for more information.
+>>> import os
+>>> os.system("sh -c \"echo 0 > /sys/class/gpio/gpio76/value\"")
```
Comment (BenGardiner) [inline README.md:57]: this isn't particularly informative though b/c it is just shelling-out the same commands.

I suggest using file.write / flush or close OR just noting that the same shell commands can get exec'd with os.system

Comment (jeremy-daily) [reply README.md:57]: I think it is important because the use of escaped quotes and the `sh -c` part is not trivial for most.

File: README.md [hunk 2]
```diff
@@ -4,6 +4,64 @@ or better yet, "can-in-the-middle" is system using the TU Truck Cape, a BeagleBo
 ## Setting up the Hardware
 The hardware for this project is 100% open. You can purchase every part of the hardware and hand assemble the pieces. The schematic for the Truck Cape or the CAN Man-in-the-middle is shown in the docs folder. There are multiple hardware versions for different form factors and switching functions. All versions use the BeagleBone Black and have 2 CAN channels and 2 J1708 channels. For more details, see the ![docs](docs) folder.
 
+### Controlling Relays
```
Comment (BenGardiner) [inline README.md:7]: neat.

It looks like there are combinations of gpio output settings that would be 'illegal' or, at least, wouldn't accomplish much: e.g. connecting CAN2 B-side to can0 and also bridging it to A side (both `echo 1 > /sys/class/gpio/gpio70/value` and `echo 1 > /sys/class/gpio/gpio76/value`)

I suggest you write a shell script or python script to capture the mapping of 'useful mode' to GPIO settings. The user will be less interested in the function of each switch and more interested in how to set the modes of operation of the cape.

Comment (jeremy-daily) [reply README.md:7]: I agree. I'll do that in a new branch.

**Pull request merged**